language-support/java/codegen/BUILD.bazel

```diff
@@ -163,55 +164,68 @@ test_daml_lf_target_versions = [
 ########################################################
 
 it_daml_lf_target_versions = [
```

Comment:
Can we share this with the LF versions in some other place? This way it’s easy to forget to support a newer version.